src/Eloquent/HybridRelations.php

```diff
@@ -41,7 +43,7 @@ trait HybridRelations
     public function hasOne($related, $foreignKey = null, $localKey = null)
     {
         // Check if it is a relation with an original model.
-        if (! is_subclass_of($related, MongoDBModel::class)) {
+        if (!is_subclass_of($related, MongoDBModel::class)) {
```

Comment:
Our CS requires that there is a space after exclamation mark. I'll revert this changes with phpcbf before merging.